### PR TITLE
minor changes to welsh content from lewis

### PR DESCRIPTION
**Pull request by GemTay**
# Purpose

Lewis has made some minor changes to the welsh content

## Checklist

* [ ] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

**Review by charlesmmarshall — DISMISSED** [review]

**Review by SeemaMenon — APPROVED** [review]

**Review by SeemaMenon — CHANGES_REQUESTED** [review]
@GemTay  - Just noticed the quotes which needs correcting

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -152,44 +152,44 @@ msgstr "Nid yw’r codau canlynol yn gweithio bellach."
...
+"If an organisation asks to see this <abbr title=\"lasting power of attorney"
+"\">LPA</abbr> again, you can <a href=\"%create-code-link%\">make them a new "
+"code</a>."
 msgstr ""
-"Os bydd sefydliad yn gofyn am gael gweld y <abbr title=\"atwrneiaeth arhosol"
-"\">LPA</abbr> hon eto, yna gallwch <a href=\"%create-code-link%\">greu cod newydd "
-"iddynt</a>."
+"Os bydd sefydliad yn gofyn am gael gweld y <abbr title=“atwrneiaeth "
+"arhosol”>LPA</abbr> hon eto, yna gallwch <a href=“%create-code-link%”>greu cod "
+"newydd iddynt</a>."
```
Comment (SeemaMenon) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:162]: The quotes have changed here

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -203,30 +203,31 @@ msgstr "Rydych wedi gofyn i ni newid eich cyfeiriad e-bost."
...
 msgid ""
-"If you’ve already confirmed the change, you can use your new email address to <a "
-"href=\"%link%\" class=\"govuk-link\">sign in</a> now."
+"If you’ve already confirmed the change, you can use your new email address to "
+"<a href=\"%link%\" class=\"govuk-link\">sign in</a> now."
 msgstr ""
-"Os ydych wedi cadarnhau’r newid yn barod, gallwch nawr ddefnyddio eich cyfeiriad e-"
-"bost newydd i <a href=\"%link%\" class=\"govuk-link\">fewngofnodi</a>."
+"Os ydych wedi cadarnhau’r newid yn barod, gallwch nawr ddefnyddio eich "
+"cyfeiriad e-bost newydd i <a href=“%link%” class=“govuk-link”>fewngofnodi</a>."
```
Comment (SeemaMenon) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:231]: Quotes changed here

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -239,7 +240,7 @@ msgstr "Diolch yn fawr. Rydym wedi cadarnhau eich cyfrif."
 #: src/Actor/templates/actor/activate-account.html.twig:22
 msgid "Please <a href=\"%link%\" class=\"govuk-link\">sign in</a>"
 msgstr ""
-"Gallwch nawr <a href=\"%link%\" class=\"govuk-link\">fewngofnodi i’ch cyfrif</a>"
+"Gallwch nawr <a href=“%link%” class=“govuk-link”>fewngofnodi i’ch cyfrif</a>"
```
Comment (SeemaMenon) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:243]: Quotes changed

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -250,8 +251,8 @@ msgid ""
 "If you already have an account, please <a href=\"%link%\" class=\"govuk-link"
 "\">sign in</a>."
 msgstr ""
-"Os oes gennych gyfrif yn barod, <a href=\"%link%\" class=\"govuk-link"
-"\">mewngofnodwch</a>."
+"Os oes gennych gyfrif yn barod, <a href=“%link%” class=“govuk-"
+"link”>mewngofnodwch</a>."
```
Comment (SeemaMenon) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:255]: Quotes

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -264,79 +265,79 @@ msgstr "Crëwch gyfrinair"
...
 #: src/Actor/templates/actor/password-change.html.twig:33
+#: src/Actor/templates/actor/password-reset.html.twig:43
 msgid "Never share your password with anyone."
 msgstr "Peidiwch â rhoi eich cyfrinair i neb arall."
 
 #: src/Actor/templates/actor/create-account.html.twig:51
 msgid "I agree to the <a href=\"%link%\" class=\"govuk-link\">terms of use</a>"
 msgstr ""
-"Rwy’n cytuno i’r <a href=\"%link%\" class=\"govuk-link\">telerau defnyddio</a>"
+"Rwy’n cytuno i’r <a href=“%link%” class=“govuk-link”>telerau defnyddio</a>"
```
Comment (SeemaMenon) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:299]: Quotes changed

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -49,7 +49,7 @@ msgstr "Iechyd a lles"
 
 #: src/Actor/templates/actor/lpa-show-viewercode.html.twig:29
 msgid "Give this access code to %organisation%"
-msgstr "Rhoi’r cod mynediad hwn i %organisation%"
+msgstr "Rhowch y cod mynediad hwn i %organisation%"
```
Comment (GemTay) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:52]: This was the only change made

Comment (cooperaj) [reply service-front/app/languages/cy/LC_MESSAGES/messages.po:52]: Aye, it's somewhat frustrating that poedit keeps messing around with the line length.

**Review by cooperaj — APPROVED** [review]

**Review by SeemaMenon — APPROVED** [review]

**Pull request merged**